.github/workflows/build_and_test.yml

```diff
@@ -41,6 +41,26 @@ jobs:
     with:
       module: save-demo-agent
       gradle-cache-read-only: ${{ github.ref != 'refs/heads/master' && github.event_name != 'pull_request' }}
+  retrieve_save_cli_version:
```

Comment:
do we removed all previous logic, corresponding for save-cli snapshots?

Reply:
yes, we refactored it during splitting save-cloud build: we had problem with long running jib on GitHub and split them to small jobs. We still have logic for save-cli in building images, but removed in regular build